examples/mps3-an536/src/bin/svc.rs

```diff
@@ -14,15 +13,14 @@ use semihosting::println;
 /// It is called by the start-up code in `cortex-m-rt`.
 #[no_mangle]
 pub extern "C" fn kmain() {
-    if let Err(e) = main() {
-        panic!("main returned {:?}", e);
-    }
+    main();
 }
 
 /// The main function of our Rust application.
 ///
 /// Called by [`kmain`].
-fn main() -> Result<(), core::fmt::Error> {
+#[export_name = "main"]
```

Comment:
I would take this out. I haven't needed it with TRACE32 or GDB.

Reply:
hmm, this is interesting.. It was important for my Zynq target, I received a really weird call stack and extremely slow stepping if main was not exported like this. I will take it out.